parcoords_translation.Rmd

```diff
@@ -0,0 +1,627 @@
+# Translation of 'parcoords' Introduction
+
+Yichi Liu and Michi Liu
+
+This is a Chinese translation version of the introduction and examples to the R package "parcoords".<br />
+这是R包“parcoords”的使用说明和示例教程的中文翻译。
+
+
+# 1. ‘parcoords’包使用说明 - 中文翻译
```

Comment:
No single `#` should be used anywhere in the file, except on the first line.  Also, there seems to be somewhat inconsistent usage of #### following ## in some places, and ### following ## in others.  In general, the sub-section of ## should be ###, not ####